state-sync/storage-service/server/src/lib.rs

```diff
@@ -365,7 +365,7 @@ impl<T: StorageReaderInterface> StorageServiceServer<T> {
                                 data_subscription,
                                 target_ledger_info,
                             ) {
-                                error!(LogSchema::new(LogEntry::SubscriptionRefresh)
+                                warn!(LogSchema::new(LogEntry::SubscriptionResponse)
```

Comment:
is this indicating malicious other actor, or can happen between two completely honest nodes?

i.e. is this indicating error in the response, or lack of receiving the response?

Reply:
This is indicating an error in the response, but it's possible in some rare cases for it to happen to two completely honest nodes, which is why I think a `warn!` makes more sense 😄 I'm not sure an `error!` is warranted, but we can monitor things.